src/model/Junction.h

```diff
@@ -43,6 +43,12 @@
  * Models a junction with arbitrary inlets and outlets. Across all inlets and
  * outlets of the junction, mass is conserved and pressure is continuous.
  *
+ * Inlets and outlets can be specified in two ways. Either using `inlet_vessels`
+ * and `outlet_vessels` keys in teh JSON file, with the corresponding lists
+ * specifying vessel IDs, or using `inlet_blocks` and `outlet_blocks` keys, with
+ * the corresponding lists specifying the names of blocks as strings.
```

Comment:
Yup, I think that's a good idea. Although I think it might be good to retain the backward compatibility as long as it isn't difficult to maintain and isn't breaking new stuff.